docs/registry/create/_index.md

```diff
@@ -600,67 +631,72 @@ func mainWithArgs(ctx context.Context, args []string, logger logger.Logger) (err
 
 </details>
 
-{{% /tab %}}
-{{< /tabs >}}
-
 {{% alert title="Important" color="note" %}}
 
 You must define all functions belonging to a built-in resource subtype's API if defining a new model.
-Otherwise, the class won’t instantiate.
+Otherwise, the class will not instantiate.
+
+If you do not wish to implement all methods, raise `errUnimplemented` for the functions you do not want to implement.
 
-- If you are using the Python SDK, raise an`NotImplementedError()` in the body of functions you do not want to implement or put `pass`.
-- If you are using the Go SDK, raise `errUnimplemented`.
-- Additionally, return any values designated in the function's return signature, typed correctly.
+Additionally, return any values designated in the function's return signature, typed correctly.
 
 {{% /alert %}}
 
+{{% /tab %}}
+{{< /tabs >}}
+
 ### Compile the module into an executable
 
-To [add a module](/modular-resources/configure/) to the configuration of your robot, you need to have an [executable](https://en.wikipedia.org/wiki/Executable) that runs your module when executed, can take a local socket as a command line argument, and cleanly exits when sent a termination signal.
+To [add a module](/registry/configure/) to the configuration of your robot, you need to have an [executable](https://en.wikipedia.org/wiki/Executable) that:
 
-Your options for completing this step are flexible, as this file does not need to be in a raw binary format.
+- runs your module when executed,
+- can take a local socket as a command line argument,
+- and cleanly exits when sent a termination signal.
+
+Your options for completing this step are flexible, as this file does not need to be in raw binary format.
 
 {{% tabs %}}
 {{% tab name="Python" %}}
 
 One option is to create and save a new shell script (<file>.sh</file>) that runs your module at your entry point (main program) file.
 
-Make sure that you set up a Python virtual environment in the directory your module is in to compile your resource properly at execution.
-See the [Python virtual environment documentation](https://docs.python-guide.org/dev/virtualenvs/) for more information.
+Make sure to set up a Python virtual environment in the directory your module is in to ensure your module has access to any required libraries:
```

Comment:
Alternatively, we could just link to this resource:

https://docs.viam.com/program/python-venv/

Reply:
that doesn't mention the `exec $PYTHON <your-src-dir-if-inside>/main.py $@` part